src/app/core/learning-object-module/search/search.routes.ts

```diff
@@ -1,8 +1,16 @@
 import { environment } from '../../../../environments/environment';
+import { Subject } from 'rxjs';
+
+const needsChange$: Subject<void> = new Subject<void>();
 
 export const SEARCH_ROUTES = {
-  GET_PUBLIC_LEARNING_OBJECTS: `${environment.apiURL}/learning-objects`,
-  GET_PUBLIC_LEARNING_OBJECTS_WITH_FILTER(query: string) {
+  needsChange$,
+  REGISTER_CHANGE() {
+    this.needsChange$.next();
+  },
```

Comment:
This should not be here, only routes to our APIs should be in the routes.ts files.

Reply:
I'm a little confused with what you mean by this. Are you asking me to move this route into the search.routehandler.ts file in clark-service?